frontend/src/pages/components/gatheredList.js

```diff
@@ -15,69 +14,66 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const colorChange = createMuiTheme({
-  palette:{
-    primary:{
-      main : "#373a3c"
-    },
-  }
-});
-
-function renderRow(props) {
-  const { index, style } = props;
-
-  return (
-    <ListItem button style={style} key={index}>
-      <ListItemText primary={`Stakeholder ${index + 1}`} />
-    </ListItem>
-  );
-}
-
-renderRow.propTypes = {
-  index: PropTypes.number.isRequired,
-  style: PropTypes.object.isRequired,
-};
-
-export default function VirtualizedList() {
+export default function InfoGatheredList({pages}) {
   const classes = useStyles();
 
-  let [showList, setShowContent] = React.useState(true);
-
+  const [showList, setShowContent] = React.useState(true);
   const toggleShow = () => {
-    setShowContent(show => !show)
+    setShowContent(show => !show);
   };
 
+  let listContentById = {};
+  let [infos, setInfos] = useContext(GatheredInfoContext);
+
+  async function getListContent(info) {
```

Comment:
As you right more complicated functions, it would be helpful to add some more detailed comments on what they do